### PR TITLE
Form redesign

### PR DESCRIPTION
**Pull request by richardwestenra**
**Updated demo here: http://richardwestenra.com/voteamerica.github.io/**

## Description

- Get rid of Bootstrap and implement slimmed-down custom form CSS.
- Redesign all the forms from the ground up.
- Keep bootstrap validator and add custom styles for it.
- Delete the existing custom validation which disabled the submit button.
- Slightly refactor most of the jQuery.

## Notes
- I'm still not completely happy with the UI design on the date/time section of the forms, but it'll do for now.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -938,168 +756,29 @@ <h3 class="bannerbox__title">Newsletter</h3>
...
-        });
-        $('#riderName').change(function() {
-            changeNeedRideSubmit();
-        });
-        $('#ridePhone').change(function() {
-            changeNeedRideSubmit();
-        });
-
-        $('.Timestamp').val(new Date());
         $('.redirect').val(window.location.origin + window.location.pathname + 'thanks.html');
```
Comment (richardwestenra) [inline index.html]: Why isn't this attribute just applied directly to the HTML? I don't understand why JS needs to be used for this?

File: index.html [hunk 2]
```diff
@@ -895,34 +727,20 @@ <h3 class="bannerbox__title">Newsletter</h3>
             });
 
             // Submit Function for needRide (Rider) form.
-            $('form[name="needRide"]').submit(function() {
+            $('form[name="needRide"]').on('submit', function() {
                 document['needRide'].action = "https://api.carpoolvote.com/v1.1/rider";
```
Comment (richardwestenra) [inline index.html]: Is these action attribute updates in the JS necessary? They're already being set in the HTML.

Comment (jkbits1) [inline index.html]: Someone else has mentioned that. Just seeing them there, I thought they must provide a handy thing to override when debugging, but it is duplication

**Comment by jkbits1** [comment]
This looks good, thanks @richardwestenra :+1:

File: index.html (hunk 2 above)
Comment (richardwestenra) [inline index.html]: Cool. I might delete them next time I'm working on the form JS.

**Pull request merged**